### ISSUE TITLE
stewards.csv always assigns "0" as the steward_id in stewards.csv

### ISSUE THREAD

**Issue by alanjosephwilliams** [codeforamerica/OpenTrails#83]
This is really a question for @migurski, whose opinions I defer to regarding the creation or approximation of IDs & UUIDs, and the practices we want to encourage in an open data specification ecosystem like OpenTrails.

Currently, the converter creates the stewards.csv file via manual input. When creating that file, the "id" column is created, and the value "0" is assigned. Across the conversion process, when each .geojson file is created, the property "steward_id" is included, and the value is written as "0".

Is this the best low level-of-effort approach to creating steward ids—which in turn, serve as a "namespacing"-type role in how data is handled (all of this geodata came from this steward, all of that geodata came from another...)? 

It seems like we should either:
1) Provide non-duplicative IDs
2) Provide guidance on why we are simply assigning the "0" value, and how to replace it before publishing your data for third-party collation with other datasets.

**Comment by alanjosephwilliams** [comment]
@migurski can you advise on this?

**Comment by migurski** [comment]
I think zeros are fine. The steward ID is local to each instance of OpenTrails data; no need to go global here.

**Comment by alanjosephwilliams** [comment]
@migurski do you think that it is the role (or responsibility) of the converter to offer guidance as to how to update it manually before publishing for 3rd-party use or combining the data with OpenTrails data from nearby jurisdictions? 

Alternatively, we  could simply make it transparent on the download page that the steward_id value is "0"—and leave the guidance as separate documentation.

**Comment by migurski** [comment]
No, I don't think it's the role of the converter. Feels like unnecessary nagging.

**Comment by alanjosephwilliams** [comment]
Sounds good. I'll call it out on the download page so it doesn't read like an error to people, but leave it at that.

**Comment by migurski** [comment]
Why would it look like an error? We can make it “1” if that looked less error-like, but why call it out?

My feeling here is that we should be trimming and focusing the feedback and hints provided to users, to avoid the “one more thing” disease of other, less-simple data formats.

**Comment by alanjosephwilliams** [comment]
Well said—for the purposes of this course, the users are interested in publishing OpenTrails an an open data set for third-party use. I'm thinking through how to set the expectations of current Converter users with the knowledge of their end goal. 

I will make the change to "1", and let's not call it out.
